docs/tutorial.md

```diff
@@ -91,31 +90,30 @@ NAME
 echo
 ```
 
-The status of type `Succeeded` shows that the `Task` ran successfully.
+The `Succeeded` status confirms that the `Task` completed with no errors.
```

Comment:
```suggestion
The `Succeeded` status confirms that the `TaskRun` completed with no errors.
```

Reply:
Again, nit, but doesn't a `TaskRun` run the `Task`?